bibliopixel/drivers/APA102.py

```diff
@@ -1,23 +1,19 @@
-from spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder
 
 
 class DriverAPA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True, dev="/dev/spidev0.0", SPISpeed=2):
-        super(DriverAPA102, self).__init__(num, c_order=c_order,
-                                           use_py_spi=use_py_spi, dev=dev, SPISpeed=SPISpeed)
+    def __init__(self, num, **kwds):
```

Comment:
Urg, it was a bad move on my part to do that, though in my defense I was trying to figure out what was getting passed around and what wasn't.

Irrelevant to the change at hand!  Dammit.

There is an issue with long repetitive names like is that when you repeat yourself a lot editing errors become much more likely, yes, but.

In this case, _nearly_ all the defaults are the same but in one or two cases they weren't.

There's got to be a better way.  I'll contemplate it.  I might throw something on top of this undoing that at the end...